boards/makerbase/mks_canable_v20/doc/index.rst

```diff
@@ -0,0 +1,176 @@
+.. zephyr:board:: mks_canable_v20
+
+Overview
+********
+
+The Makervase MKS CANable V2.0 board features an ARM Cortex-M4 based STM32G431C8 MCU
+with a wide range of connectivity support and configurations.
+Here are some highlights of the MKS CANable V2.0 board:
+
+- STM32 microcontroller in LQFP48 package
+- USB Type-C connector (J1)
+- CAN-Bus connector (J2)
+- ST-LINK/V3E debugger/programmer header (J4)
+- USB VBUS power supply (5 V)
+- Three LEDs: red/power_led (D1), blue/stat_led (D2), green/word_led (D3)
+- One push-button for RESET
+
+The LED red/power_led (D1) is connected directly to on-board 3.3 V and not controllable by the MCU.
+
+More information about the board can be found at the `MKS CANable V2.0 website`_.
+It is very advisable to take a look in on user manual `MKS CANable V2.0 User Manual`_ and
+schematic `MKS CANable V2.0 schematic`_ before start.
+
+- Development support: serial wire debug (SWD), JTAG, Embedded Trace Macrocell.
```

Comment:
This bullet point looks isolated. Should it be grouped in above list ? Or maybe rephrased ?